src/main/java/com/example/java/maven/cardGamePeterPan/Game.java

```diff
@@ -97,35 +93,31 @@ public Player getPreviousPlayer(Player player) {
         }
     }
 
-    public int countActivePlayers() {
-        int activePlayerCounter = 0;
-        for (Player player : playerList) {
-            if (player.getHandSize() > 0) {
-                activePlayerCounter += 1;
-            }
-        }
-        return activePlayerCounter;
+    public int numberOfActivePlayers() {
+        return getActivePlayers().size();
     }
 
     public List<Player> getActivePlayers(){
         return playerList.stream()
-                .filter(player -> player.getHandSize() > 0)
+                .filter(this::playerIsActive)
                 .collect(Collectors.toList());
     }
 
-    public boolean checkIfPlayerIsActive(Player player) {
+    public boolean playerIsActive(Player player) {
         return player.getHandSize() != 0;
     }
 
     public void play() {
-        while (countActivePlayers() > 1) {
+        while (numberOfActivePlayers() > 1) {
             for (Player player : getActivePlayers()) {
-                if (checkIfPlayerIsActive(player)) {
-                    messagePrinter.printMessage(String.format("Your turn, %s", player.getName()));
-                    getCardFromPreviousPlayerHand(player);
-                    if (player.isDiscardingCardsPossible()) {
-                        choseCardsToDiscard(player);
-                    }
+                messagePrinter.printMessage(String.format("Your turn, %s", player.getName()));
+                getCardFromPreviousPlayerHand(player);
+                if (player.isDiscardingCardsPossible()) {
+                    discardCards(player);
+                }
+                if (numberOfActivePlayers() == 1) {
```

Comment:
not sure that this if is better than doing this logic outside of the `while` loop :D